### PR TITLE
Fix the path for the ajax-loader.gif

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue #15305 and altornative to #15529 .

### Summary of Changes
Fix the path for the ajax-loader.gif

### Testing Instructions
Please see #15305 and #15529

### Documentation Changes Required
none

### CODE REVIEW

File: media/system/js/core-uncompressed.js [hunk 1]
```diff
@@ -730,10 +730,11 @@ Joomla.editors.instances = Joomla.editors.instances || {
...
 
 		// Create the loading layer (hidden by default).
-		if (task == 'load')
+		if (task === 'load')
 		{
-			// Gets the site base path from the body element (defaults to empty - no subfolder)
-			var basePath = document.getElementsByTagName('body')[0].getAttribute('data-basepath') || '';
+			// Gets the site base path
+			var systemPaths = Joomla.getOptions('system.paths') || {},
+				basePath    = systemPaths.root || '';
```
Comment (Quy) [inline media/system/js/core-uncompressed.js:742]: Remove extra spaces.

**Comment by franz-wohlkoenig** [comment]
Tested with and -out Menu "Create Article" and with and -out PR got no Error in Console>JS.
### System information
3.7.3-dev (latest nightly build)
Multilanguage Site
macOS Sierra, 10.12.5
Firefox 54 (64-bit)
#### MAMP 4.1.1
- PHP 7.0.15
- MySQLi 5.6.35

File: media/system/js/core-uncompressed.js (hunk 1 above)
Comment (Fedik) [reply media/system/js/core-uncompressed.js:742]: in the editor it looks perfect :neckbeard: 
screen 2017-06-15 20 35 24 725x120

Comment (Quy) [reply media/system/js/core-uncompressed.js:742]: It looks off in the GitHub UI and in my editor with downloaded raw version.

spaces

**Comment by Quy** [comment]
@franz-wohlkoenig Without the pr, I get the error when editing an article in the frontend.

**Comment by infograf768** [comment]
Firefox
I do not get the error either.
Is the ajax-loader.gif supposed to be displayed when saving such an article?
The only error I get is:
`Empty string passed to getElementById().jquery.min.js:2:24932`

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 502183a17cd8606ee57650aefc2da27546a728e6<br /><br />Safari displays the error. Patch solves the issue.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16701">issues.joomla.org/tracker/joomla-cms/16701</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 502183a17cd8606ee57650aefc2da27546a728e6<br /><br />Didn't get Error in Safari on Mac, but on Chrome.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16701">issues.joomla.org/tracker/joomla-cms/16701</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by mbabker** [comment]
Merge conflicts need to be resolved.

**Pull request merged**